apps/src/applab/gridUtils.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * A couple of utility functions for dealing with our design mode grid.
+ */
+module.exports = {
+  GRID_SIZE: 5,
+  /**
+   * Given an element being dragged, determine the scaled x/y position of the
+   * top left corned, scaled to our visualization.
+   * @param {jQueryObject} draggingElement
+   * @return {object}
+   */
+  scaledDropPoint: function (draggingElement) {
+    var div = document.getElementById('designModeViz');
```

Comment:
Usage of this in VisualizationOverlay also means it now has a dependency on there being this element in the dom named designModeViz. What are your thoughts on this? Could pass it in as a param, but seems a little silly when we're always using this element.